### DISCUSSION TITLE
Delay in getting Student Developer pack benefits

### DISCUSSION THREAD

**Discussion by soumik2024** [community/community#135158] [GitHub Education]
### Hi everyone,

It has been 4 days since My application is accepted, still I haven't got the benefits of student developer pack yet. When will I get it?

![Screenshot 2024-08-08 232001](https://github.com/user-attachments/assets/495042b5-7ae1-4f45-b9c5-4e582f232d22)

**Comment by davevad93** [comment]
Hi @soumik2024 ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @soumik2024 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
